shacl12-overview/index.html

```diff
@@ -424,7 +427,7 @@ <h2>What's New in SHACL 1.2</h2>
                 The new <em>Inferencing Rules</em> specification allows the representation and execution of <a href="https://en.wikipedia.org/wiki/Datalog">Datalog</a>-like inference rules which have a straight-forward mapping to SPARQL <code>CONSTRUCT</code> statements for execution, and integrate with other SHACL features such as node expressions.
               </p>
               <p>
-                This work was inspired by the SPARQL-based rules introduced by the <a href="https://www.w3.org/TR/shacl-af/">SHACL 1.1 Advanced Features</a> note (commonly referred to as "SHACL-AF") and older technologies such as <a href="https://spinrdf.org/">SPIN</a>.
+                This work was inspired by the SPARQL-based rules introduced by the <a href="https://www.w3.org/TR/shacl-af/">SHACL Advanced Features</a> note (commonly referred to as "SHACL-AF") and older technologies such as <a href="https://spinrdf.org/">SPIN</a>.
```

Comment:
```suggestion
                This work was inspired by the SPARQL-based rules introduced by the <a href="https://www.w3.org/TR/2017/NOTE-shacl-af-20170608/">SHACL Advanced Features</a> note (commonly referred to as "SHACL-AF") and older technologies such as <a href="https://spinrdf.org/">SPIN</a>.
```

Again, I'd prefer the dated-URL permalinks, but I won't push so much for this instance if you disagree.  NACK'ing here is easily fine with me.